daphne/src/lib.rs

```diff
@@ -48,6 +48,10 @@ pub enum DapError {
     #[error("fatal error: {0}")]
     Fatal(String),
 
+    /// Error caused due to messages from a peer.
+    #[error("peer error: {0}")]
+    PeerError(String),
```

Comment:
If you do this instead, you can get rid of the hard-coded stuff. By the way, you're going to need to add a new abort type: See the changes in pr #297 on the DAP spec.
```suggestion
    PeerError(DapAbort),
```

Reply:
We are now using DapAbort instead of String.